specs/player_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+require_relative '../lib/player'
+require_relative '../lib/scoring'
+require_relative '../lib/tile_bag'
+require 'pry'
+
+Minitest::Reporters.use! Minitest::Reporters::SpecReporter.new
+
+describe 'Player' do
+  before do
+    @player = Scrabble::Player.new("Paul")
+  end
+
+  describe 'initialize' do
+    it "Takes a name" do
+      @player.must_respond_to :name
+      @player.name.must_equal "Paul"
+    end
+  end
+
+  describe 'plays' do
+    it "Returns an array of the words played by the player" do
+      @player.play("banana")
+      @player.play("orange")
+
+      @player.plays.must_equal ["banana", "orange"]
+    end
+
+    it "Returns an empty array if the player hasn't played" do
+        @player.plays.must_equal []
+    end
+  end
+
+  describe 'play' do
+    it "Adds the input word to the `plays` array" do
+      @player.play("banana")
+      @player.play("orange")
+
+      @player.plays.include?("banana").must_equal true
+      @player.plays.include?("orange").must_equal true
+      @player.plays.length.must_equal 2
+    end
+
+    it "Returns `false` if player has already won" do
+      @player.play("zzzzzzz")
+
+      @player.play("banana").must_equal false
+    end
+
+    it "Returns the score of the `word`" do
+      @player.play("banana").must_equal 8
+    end
+
+    it "Returns ArgumentError if the word is already present" do
+      @player.play("banana")
+
+      proc {@player.play("banana")}.must_raise ArgumentError
+    end
+  end
+
+  describe 'total_score' do
```

Comment:
What happens to the score if they've won and you try to play another word?  The score shouldn't change.  You should **test** that!